website/content/sdk/quickstart/pypi.md

```diff
@@ -0,0 +1,50 @@
+[![Downloads](https://static.pepy.tech/badge/openbb)](https://pepy.tech/project/openbb)
+[![LatestRelease](https://badge.fury.io/py/openbb.svg)](https://github.com/OpenBB-finance/OpenBBTerminal)
+
+| OpenBB is committed to build the future of investment research by focusing on an open source infrastructure accessible to everyone, everywhere. |
+|:--:|
+| <img width="1000" alt="Screenshot 2023-02-14 at 5 12 08 PM" src="https://user-images.githubusercontent.com/25267873/218899768-1f0964b8-326c-4f35-af6f-ea0946ac970b.png"> | 
+| Check our website at [openbb.co](www.openbb.co) |
+
+
+## OpenBB SDK Overview
+
+OpenBB SDK provides a convenient way to access raw financial data from multiple data providers. This allows users to build custom financial dashboards and reports in minutes.
+
+More information on this product can be found [here](https://openbb.co/products/sdk)
+
+
+## Installation
+
+The command below provides access to the core functionalities behind the [OpenBB Terminal](https://openbb.co/products/terminal).
+
+```python
+pip install openbb
+```
```

Comment:
This goes against what we mention in the docs since we mention to use Miniconda and install an environment specifically. This could result in weird bugs when people do not expect our package to bring all these kind of packages. Generally, when you install something like Pandas this is all good, for us it isn't. See: https://docs.openbb.co/sdk/quickstart/installation